src/main/java/thePackmaster/relics/summonspack/BlueSkull.java

```diff
@@ -0,0 +1,70 @@
+package thePackmaster.relics.summonspack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.RelicAboveCreatureAction;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.FocusPower;
+import com.megacrit.cardcrawl.rooms.AbstractRoom;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.relics.AbstractPackmasterRelic;
+
+import static thePackmaster.util.Wiz.adp;
+import static thePackmaster.util.Wiz.att;
+
+public class BlueSkull extends AbstractPackmasterRelic {
+    public static final String ID = SpireAnniversary5Mod.makeID(BlueSkull.class.getSimpleName());
+    private static final int FOCUS_AMOUNT = 2;
+    private boolean isActive = false;
+
+    public BlueSkull() {
+        super(ID, RelicTier.COMMON, LandingSound.FLAT);
+    }
+
+    public void atBattleStart() {
+        isActive = false;
+        addToBot(new AbstractGameAction() {
+            public void update() {
+                if (!isActive && adp().isBloodied) {
+                    flash();
+                    pulse = true;
+                    adp().addPower(new FocusPower(adp(), FOCUS_AMOUNT));
+                    att(new RelicAboveCreatureAction(adp(), BlueSkull.this));
+                    isActive = true;
+                    AbstractDungeon.onModifyPower();
+                }
+                isDone = true;
+            }
+        });
+    }
+
+    public void onBloodied() {
+        flash();
+        pulse = true;
+        if (!isActive && AbstractDungeon.getCurrRoom().phase == AbstractRoom.RoomPhase.COMBAT) {
+            att(new ApplyPowerAction(adp(), adp(), new FocusPower(adp(), FOCUS_AMOUNT), FOCUS_AMOUNT));
+            att(new RelicAboveCreatureAction(adp(), this));
+            isActive = true;
+            adp().hand.applyPowers();
+        }
+
+    }
+
+    public void onNotBloodied() {
+        if (isActive && AbstractDungeon.getCurrRoom().phase == AbstractRoom.RoomPhase.COMBAT)
```

Comment:
```suggestion
        if (isActive && Wiz.isInCombat())
```
Current check can crash theoretically